KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutPresenter.swift

```diff
@@ -67,7 +71,7 @@ final class KarhooCheckoutPresenter: CheckoutPresenter {
         if karhooUser {
             completeLoadingViewForKarhooUser(view: view)
         }
-        view.set(quote: quote)
+        view.set(quote: quote, loyaltyInfo: loyaltyInfo ?? LoyaltyInfo(canEarn: false, canBurn: false))
```

Comment:
If the loyaltyInfo is null, maybe we can just hide the whole view rather than passing LoyaltyInfo(canEarn:false, canBurn: false)

Reply:
done